doc/classes/SpatialMaterial.xml

```diff
@@ -201,10 +240,13 @@
 			The strength of the normal map's effect.
 		</member>
 		<member name="normal_texture" type="Texture" setter="set_texture" getter="get_texture">
+			Texture used to specify the normal at a given pixel. [normal_texture]s only use the red and green channels. The normal read from [normal_texture] is oriented around the surface normal provided by the [Mesh].
```

Comment:
The trick that `[Self]` becomes `[code]Self[/code]` is only valid for the class, not for member variables :)